data-raw/fft.R

```diff
@@ -1,10 +1,84 @@
 ## code to prepare `fft` dataset goes here
 
+library(tidyverse)
```

Comment:
I would generally avoid importing the entire tidyverse because that will often import a lot of packages that you won't use in your code. 

In this case I think it's better to avoid importing any libraries at all (calling functions directly instead, using the double-colon operator, `::`), but generally I would recommend calling the specific packages you need.

Reply:
Tidyverse call has been removed and double-colon operator has been added wherever it is missing.